Sources/TuistSupport/Extensions/String+Extras.swift

```diff
@@ -193,6 +193,8 @@ public extension Array where Element: CustomStringConvertible {
     /// ["Framework", "App", "Tests"] results in "Framework, App, and Tests"
     /// - Returns: <#description#>
     func listed() -> String {
-        ListFormatter().string(from: self) ?? ""
+        let l = ListFormatter()
```

Comment:
nitpick: I'd just name this `listFormatter` instead of one letter

Reply:
Oops, good catch.